usaspending_api/api_contracts/contracts/v1/references/filter.md

```diff
@@ -0,0 +1,131 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Advanced Search
+
+These endpoints support the advanced search page and allow for complex filtering for specific subsets of spending data.
+
+# Group Visualizations
+
+These endpoints return data that are grouped in preset units to support the various data visualizations on USAspending.gov's Advanced Search page.
+
+## Generated Filter Hash for URL [/api/v1/references/filter/]
+
+Generates a hash for URL, based on selected filter criteria.
+
+### Generated Filter Hash for URL Data [POST]
++ Request (application/json)
+    + Attributes (object)
+        + filters (optional, FilterObject)
+
++ Response 200 (application/json)
+    + Attributes
+        + hash : `5703c297b902ac3b76088c5c275b53f9` (required, string)
+
+# Data Structures
+
+## Filter Objects
+### FilterObject (object)
++ `keywords` : poptarts (optional, array[string])
```

Comment:
The request is formatted like the filter object in the `v1/references/hash/` contract response. 
```
"awardAmounts": {},
"awardType": [],
"extentCompeted": [], ...
```

Reply:
Neither of them actually use the FilterObject mentioned in the "Data Structures" section?

Reply:
This is an old version of the API